pwndbg/gdblib/arch.py

```diff
@@ -82,6 +82,10 @@ def _get_arch(ptrsize: int):
     else:
         endian = "big"
 
+    # Importing requires that `pwndbg.dbg` already be set up, so we have to do
+    # it here, rather then on the top level.
+    import pwndbg.gdblib.proc
```

Comment:
Why is there this requirement?

Reply:
`pwndbg.dbg.gdb` needs `pwndbg.gdblib`, which imports `pwndbg.gdlib.arch` on the top-level, which needs `pwndbg.gdblib.proc`, which needs `pwndbg.gdblib.qemu`, which uses the event handler. Since the event handler now needs `pwndbg.dbg` to be set up, we can't have that import on the top level.